pkgs/development/ocaml-modules/mirage-flow/unix.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, buildDunePackage, fmt, logs, mirage-flow, ocaml_lwt, cstruct
+, alcotest, mirage-flow-combinators }:
+
+buildDunePackage {
+  pname = "mirage-flow-unix";
+
+  inherit (mirage-flow) version src;
+
+  propagatedBuildInputs = [ fmt logs mirage-flow ocaml_lwt cstruct ];
+
+  doCheck = true;
+  checkInputs = [ alcotest mirage-flow-combinators ];
+
+  meta = mirage-flow.meta // {
+    description = "Flow implementations and combinators for MirageOS specialized to lwt";
```

Comment:
```suggestion
    description = "Flow implementations and combinators for MirageOS on Unix";
```

Reply:
Yeah, that's a copy-paste mistake, fixed.